semgrep-core/tests/lua/assignment.lua

```diff
@@ -0,0 +1,19 @@
+-- ERROR: match
+foo = foo
+
+foo = {}
+
+-- ERROR: match
+foo = foo
+
+qux = foo:bar() == foo:baz()
+
+qux = foo:bar() == foo:bar()
+
+local a, b, c = 1, 2, 3
+
+-- ERROR: match
+a, b, c = a
```

Comment:
This should not match, unless I don't understand the semantic of equal in Lua.
Is it equivalent to  OCaml let (a,b,c) = a? 
In that case it should be also transformed into an Assign (Tuple (a,b,c), a)

Reply:
No, `a, b = 1, 2` is the equivalent of `a = 1; b= 2`. It's similar to Go.